app/middleware/guide-container.js

```diff
@@ -1,8 +1,9 @@
 module.exports = (req, res, next) => {
   const pageData = req.pageData || {};
+  const children = pageData.meta ? pageData.meta.children : [];
 
-  if (pageData.layout === 'guide') {
-    const first = pageData.pages[0];
+  if (pageData.guide && children && children.length > 0) {
```

Comment:
You might have to check if it's a folder type as well as we might have a guide with more levels.
e.g.:
```
folder guide
    page guide
    page guide
        page guide
```

We can ignore this case for now though.